task-reminder-client/libs/pages/login/src/lib/login/login.component.scss

```diff
@@ -0,0 +1,9 @@
+.login {
+  background: url('../../../../../../apps/task-reminder/src/assets/images/background-login.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.logo img {
+  content: url('../../../../../../apps/task-reminder/src/assets/images/waverley_logo.png');
```

Comment:
use **src** property in HTML for img tag.
if you want to define styles for child tag, use sass syntax. Ex:
```
.logo {
   img {

   }
}
```